1528.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param {string} s
+ * @param {number[]} indices
+ * @return {string}
+ */
+var restoreString = function (s, indices) {
+  const sorted = indices.slice();
+  let result = [];
+
+  for (let i = 0; i < indices.length; i++) {
+    result[sorted[i]] = s[i];
```

Comment:
```suggestion
    result[indices[i]] = s[i];
```
You can just use `indices` instead of `sorted`